test/unit/basic.js

```diff
@@ -0,0 +1,268 @@
+module( "basic", {
+	teardown: moduleTeardown
+} );
+
+if ( jQuery.ajax ) {
+test( "ajax", function( assert ) {
+	expect( 4 );
+
+	var done = assert.async(),
+		requestsLeft = 3;
+
+	jQuery.ajax({
+		type: "GET",
+		url: url( "data/name.php?name=foo" ),
+		success: function( msg ) {
+			assert.strictEqual( msg, "bar", "Check for GET" );
+			requestsLeft--;
+			if ( !requestsLeft ) {
+				done();
```

Comment:
Of course you're right about `.call()`, and I agree on the former point. In fact, I meant to create a ticket for that earlier, but it fell off my radar. Anyway, corrected now: jquery/qunit/issues/843